### DISCUSSION TITLE
Jupyter in VScode server

### DISCUSSION THREAD

**Discussion by phisanti** [coder/code-server#2645] [Q&A] [answered]
I would to know if there is any dockerfile command or any other trick we can use to install jupyter extension.

**Comment by code-asher** [comment]
There are some workarounds here you can check out: https://github.com/cdr/code-server/issues/2341

**Comment by nooobcoder** [comment, marked as answer]
@phisanti  You can download a working version of the working Python extension here
[https://www.vsixhub.com/go.php?post_id=39743&s=private&link=https%3A%2F%2Ff.vsixhub.com%2Ffile.php%3Fpost_id%3D39743%26app_id%3Df1f59ae4-9318-4f3c-a9b5-81b2eaa5f8a5%26version%3D2020.5.86806%26ext_name%3Dpython](url)

_SIDENOTE_

After downloading, paste the **_.vsix_** file to a mounted location on the server, where it would be accessible. I use a docker container and i bind my volumes to have persistent data. If you do not use docker then you should be good to go already

Just install it from this option
Go to the extensions tab and right click the three dots on the top 
![image](https://user-images.githubusercontent.com/50350828/107322793-27965100-6acb-11eb-9d95-a7bf899f7721.png)
then hit **Install from VSIX** from there select the .vsix (extension) you downloaded and VOILA! you should be good to go. I personally worked with this version, everything works just GRREATTT!


**If your query has been answered, please mark this thread as answered.**
Thank you,
HAVE A GREAT DAT @phisanti

**Reply by phisanti** [reply]
The link there seems to be broken, would you mind to paste it again?

**Reply by nooobcoder** [reply]
@phisanti https://www.vsixhub.com/vsix/39743/
Get it from here
![image](https://user-images.githubusercontent.com/50350828/107343583-79e56b00-6ae7-11eb-9a7b-418206f5c143.png)

Click on the option that says, download from **VSIXHUB**

**Reply by phisanti** [reply]
Thanks! I have installed and I can run it with no problem. No ipywidgets tho! I will keep using it until we can get the jupyter/python separate extensions.

**Reply by nooobcoder** [reply]
@phisanti Please mark the thread as answered. It would make a contribution to me as a student.
Hope you have a great day.
Thank You
